### DISCUSSION TITLE
What does the step "The resulting NvStrapsReBar.ffs file needs to be included in the motherboard UEFI image" mean?

### DISCUSSION THREAD

**Discussion by Qewar** [terminatorul/NvStrapsReBar#58] [Q&A] [answered]
How exactly is it included? Do I just need to place the .ffs file alongside the motherboard UEFI image? I doubt it works like that, but the steps are quite vague. Is there another guide I could follow that doesn’t require me to hop between repositories to follow the steps?

**Comment by terminatorul** [comment, marked as answer]
Follow the instructions from the [ReBarUEFI](https://github.com/xCuri0/ReBarUEFI) project, namely this page:
https://github.com/xCuri0/ReBarUEFI/wiki/Adding-FFS-module

Reading the documentation from another repository can be good to avoid duplicating the tutorial, and have one official source of documentation.
